install.ps1

```diff
@@ -108,6 +112,8 @@ if (($exchserver.admindisplayversion).major -eq 8 -and ($exchserver.admindisplay
 	$EXVER="Exchange 2016 CU6"
 } elseif (($exchserver.admindisplayversion).major -eq 15 -and ($exchserver.admindisplayversion).minor -eq 1 -and ($exchserver.admindisplayversion).build -eq 1261) {
 	$EXVER="Exchange 2016 CU7"
+} elseif (($exchserver.admindisplayversion).major -eq 15 -and ($exchserver.admindisplayversion).minor -eq 1 -and ($exchserver.admindisplayversion).build -eq 1415) {
+	$EXVER="Exchange 2016 CU7"
```

Comment:
Should this be $EXVER="Exchange 2016 CU8"

Reply:
Yes it should. Thanks for spotting that.

Reply:
Fixed a typo in README.md as well.